diff --git a/data/org.gnome.Cheese.appdata.xml.in b/data/org.gnome.Cheese.appdata.xml.in
index 19c7b5d..f76c0c2 100644
--- a/data/org.gnome.Cheese.appdata.xml.in
+++ b/data/org.gnome.Cheese.appdata.xml.in
@@ -37,4 +37,10 @@
   <update_contact>davidk@gnome.org</update_contact>
   <project_group>GNOME</project_group>
   <translation type="gettext">cheese</translation>
+  <content_rating type="oars-1.1"/>
+  <releases>
+    <release version="3.38.0" date="2020-09-16">
+      <url>https://gitlab.gnome.org/GNOME/cheese/raw/3.38.0/NEWS</url>
+    </release>
+  </releases>
 </component>
